pkg/ruler/api_test.go

```diff
@@ -1071,7 +1124,7 @@ rules:
 
 				router.ServeHTTP(w, req)
 				require.Equal(t, 200, w.Code)
-				require.YAMLEq(t, tt.output, w.Body.String())
+				require.YAMLEq(t, tt.input, w.Body.String())
```

Comment:
Note to reviewers: we can simplify the test. We expect to receive the same YAML we uploaded.

Reply:
agree, although isn't that because we don't test 0 value here?